modules/covid_research/lib/redis_format.rb

```diff
@@ -2,8 +2,6 @@
 
 module CovidResearch
   class RedisFormat
-    attr_reader :iv
```

Comment:
Where was this `iv` being used? I'm only seeing the test changes in this PR. What was the iv used for here? Do we need to replicate it in lockbox?

Reply:
No, lockbox doesn't use an iv. iv was passed to the encrypt/decrypt methods in the modules/covid_research/app/services/covid_research/volunteer/form_crypto_service.rb (see the changes made to that file in this PR)

Reply:
Reading the lockbox docs, Lockbox uses a unique key per field via CipherSweet.

Reply:
@ericboehs Related to persisting encrypted data in redis, the encryption logic is only intended to be used for form data when it's put into Redis (for Sidekiq), so it doesn't appear that we need to be concerned with additional encrypted data in Redis.

Reply:
Lockbox does use an iv, but it's now stored in one field instead of two. See ["Cleaner Schema"](https://ankane.org/modern-encryption-rails) heading in the docs.